storybook/webpack.config.js

```diff
@@ -120,5 +120,6 @@ module.exports = async ({ config }) => {
     '@elastic/charts/': path.resolve(__dirname, '../packages/charts/'),
   };
 
+  // eslint-disable-next-line @typescript-eslint/return-await
   return await config;
```

Comment:
remove `async` from function expression too

```suggestion
  return config;
```